src/components/editor/InputLiteral.jsx

```diff
@@ -3,9 +3,8 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+// import { getProperty } from '../../reducers/index'
```

Comment:
same thing about commented out line -- can it be removed altogether, or should it be mentioned in a ticket somewhere?